modules/update-manifest-application.adoc

```diff
@@ -43,12 +43,16 @@ This is especially true for cluster Operators, which might perform their own upd
 While the additional update actions take place, these cluster Operators temporarily set their `Progressing` condition to `True`.
 ====
 
+// to do: potentially reword the note above to clarify that specific resources are being applied at one time, and not necessarily all the resources for that component.
```

Comment:
This is in reference to [this discussion thread](https://github.com/openshift/openshift-docs/pull/60091#discussion_r1212371011).